jelares/portfolio/target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Mon Jun 22 16:57:27 UTC 2020
+#Tue Jun 23 16:01:07 UTC 2020
```

Comment:
I'm wondering, can this file be added to .gitignore so that this type of change doesn't show up in PRs?